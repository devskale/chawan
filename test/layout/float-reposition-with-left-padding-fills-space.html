<!DOCTYPE html>
<div style="padding: 0 1ch; background: purple">
<div style="float: right">
</div>
<div style="background: red">
<div style="background: blue; display: flow-root">
test test test
