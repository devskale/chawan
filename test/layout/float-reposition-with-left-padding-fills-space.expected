[48;2;128;0;128m [48;2;0;0;255mtest test test                                                                [48;2;128;0;128m [49m
